README.md

```diff
@@ -1,2 +1,21 @@
 # hw01
-Repository for homework 01
+
+
+# Hello, my name is Yanting Wu
+
+## I am a MAPSS student studying Psychology 
+
+*I am originally from China*
+
+__*I double majored in Psychology and Statistics when I was an undergraduate student at the University of Virginia*__
+
+**Here are some random fun facts about me**
+1. I have a golden retriever puppy 
+1. I have a younger sister 
+1. My interests lie in social psychology
+
+![GitHub Logo](mydogandI.JPG)
+
+
+Here is the instagram account for my dog: 
+[Jamie_the_goldengirl](https://www.instagram.com/jamie_the_goldengirl/?hl=en)
```

Comment:
Did you mean this to appear on a new line? If so, add a blank line between them. For instance:

```
Here is the instagram account for my dog: 

[Jamie_the_goldengirl](https://www.instagram.com/jamie_the_goldengirl/?hl=en)
```

Here is the instagram account for my dog: 

[Jamie_the_goldengirl](https://www.instagram.com/jamie_the_goldengirl/?hl=en)